firmware/hw_layer/drivers/gpio/tle6240.cpp

```diff
@@ -218,7 +218,7 @@ int Tle6240::chip_init()
 	}
 
 	/* release reset */
-	if (!cfg->reset.port) {
+	if (cfg->reset.port) {
```

Comment:
oops

Reply:
I know that I should review and test this last weekend...

Reply:
I thought andrey was going to wait to merge until you'd tested, but then I got email this morning it had merged :joy: